### PR TITLE
Adding our standard Rubocop style

### PR DESCRIPTION
**Pull request by wfleming**
pushing so I can see the issue diff from CC: not ready for review yet.

### CODE REVIEW

**Comment by wfleming** [comment]
Issues addressed, everything is green, quick look @codeclimate/review?

File: Gemfile.lock [hunk 1]
```diff
@@ -1,14 +1,14 @@
 PATH
   remote: .
   specs:
-    codeclimate-yaml (0.2.3)
+    codeclimate-yaml (0.3.0)
```
Comment (GordonDiggs) [inline Gemfile.lock]: Were these version bumps intentional?

**Comment by pbrisbin** [comment]
Same question re version bump. LGTM besides.

File: Gemfile.lock (hunk 1 above)
Comment (wfleming) [inline Gemfile.lock]: 7fdcc53:
> Update Gemfile.lock …
> Looks like this didn't get committed last time the gem was bumped.

**Pull request merged**